src/components/graph/graph.helper.js

```diff
@@ -451,11 +451,23 @@ function updateNodeHighlightedValue(nodes, links, config, id, value = false) {
     };
 }
 
+/**
+ *
+ * @param {Object.<number, number>} vector a 2D vector with x and y components
```

Comment:
Just `@param {Object}`, it's not a map alike object, it has two properties `x` and `y`.